<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <bold>false</bold>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>PCP Fabricacao</string>
  </property>
  <property name="iconSize">
   <size>
    <width>20</width>
    <height>20</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>480</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>640</width>
     <height>480</height>
    </size>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>620</width>
      <height>300</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>620</width>
      <height>300</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>620</width>
      <height>300</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_Gerador">
     <attribute name="title">
      <string>Gerador de pastas</string>
     </attribute>
     <widget class="QGroupBox" name="GrupoBotoesAdicionais">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>170</y>
        <width>120</width>
        <height>90</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton_LimparTudo">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>15</y>
         <width>90</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Limpar Tudo</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_reserva">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>50</y>
         <width>90</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="GrupoLocalDesenhos">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>460</width>
        <height>70</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_Desenhos">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>5</y>
         <width>221</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Local dos Desenhos (PDF_DWG_IGS)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_ProcurarDesenhos">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>30</y>
         <width>90</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Procurar</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Desenhos">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>340</width>
         <height>25</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="GrupoReserva">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>325</x>
        <y>170</y>
        <width>145</width>
        <height>90</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_Res">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>5</y>
         <width>130</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Pastas</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_Processos">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Processos</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_Res2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="GrupoBotoesGerador">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>10</y>
        <width>120</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton_GerarPastas">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>10</y>
         <width>90</width>
         <height>60</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>GERAR
Pastas com
desenhos</string>
       </property>
       <property name="shortcut">
        <string>Return</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Atualizar">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>80</y>
         <width>90</width>
         <height>60</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>ATUALIZAR
Deleta tudo
e refaz</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="GrupoCaminho">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>460</width>
        <height>70</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_ArquivoPCP">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>5</y>
         <width>80</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Arquivo PCP</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_ProcurarPCP">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>30</y>
         <width>90</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Procurar</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_ArquivoPCP">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>340</width>
         <height>25</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="GrupoArquivos">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>150</width>
        <height>90</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_GrupoArquivos">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>5</y>
         <width>130</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nome dos arquivos</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_codigo">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Código</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_referencia">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Referência</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="GrupoPastas">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>170</y>
        <width>145</width>
        <height>90</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_Pastas">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>5</y>
         <width>130</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Pastas Adicionais</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_Pintura">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Pintura</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_Usinagem">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Usinagem</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_ImportarLista">
     <attribute name="title">
      <string>Importar CSV</string>
     </attribute>
     <widget class="QGroupBox" name="GrupoPMS">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>460</width>
        <height>70</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_ArquivoPMS">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>5</y>
         <width>271</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Arquivo PMS com a lista para fabricacao</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_ArquivoPMS">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>30</y>
         <width>90</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Procurar</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_ArquivoPMS">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>340</width>
         <height>25</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="GrupoArquivoPCPPadrao">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>460</width>
        <height>70</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_NomeArquivo">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>5</y>
         <width>231</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Arquivo PCP Padrão</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_ArquivoPCPPadrao">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>30</y>
         <width>90</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Procurar</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_ArquivoPCPPadrao">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>340</width>
         <height>25</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="GrupoBotoesCSV">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>10</y>
        <width>120</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton_GerarPCP">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>10</y>
         <width>90</width>
         <height>60</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>CRIAR
Planilha e
pasta de PCP</string>
       </property>
       <property name="shortcut">
        <string>Return</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_VerificarArquivos">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>80</y>
         <width>90</width>
         <height>60</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>VERIFICAR
arquivos
faltantes</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="GrupoGerarNomes">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>220</width>
        <height>70</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_NomeProjeto">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>50</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Projeto</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_NomeProjeto">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>5</y>
         <width>150</width>
         <height>25</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_NomeLista">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>35</y>
         <width>150</width>
         <height>25</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_NomeLista">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>50</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Lista</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QTextEdit" name="StatusPanel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>620</width>
      <height>140</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>620</width>
      <height>140</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
